docs/content/en/docs/reference/changelog.md

```diff
@@ -6,10 +6,19 @@ weight: 35
 
 ## Unreleased
```

Comment:
Can we delete "## Unreleased" ? I think it showed up before and we got rid of it.

Reply:
I think we usually remove it when we back port to the release branch? But main retains it?

Reply:
This is for the main branch (it acts as staging) so once we bring the commit to the release branch (which will be published), then it won't show. I think before we accidentally brought the unreleased section to the release branch.

Reply:
Got it. Thanks.